tools/process_headers.zig

```diff
@@ -31,31 +31,31 @@ const glibc_targets = [_]LibCTarget{
     .{ .name = "armeb-linux-gnueabihf", .arch = .armeb, .abi = .gnueabihf },
     .{ .name = "aarch64-linux-gnu", .arch = .aarch64, .abi = .gnu },
     .{ .name = "aarch64_be-linux-gnu", .arch = .aarch64_be, .abi = .gnu },
-    .{ .name = "csky-linux-gnuabiv2-soft", .arch = .csky, .abi = .gnueabi },
-    .{ .name = "csky-linux-gnuabiv2", .arch = .csky, .abi = .gnueabihf },
-    .{ .name = "loongarch64-linux-gnu-lp64d", .arch = .loongarch64, .abi = .gnu },
-    .{ .name = "loongarch64-linux-gnu-lp64s", .arch = .loongarch64, .abi = .gnusf },
+    .{ .name = "csky-linux-gnueabi", .arch = .csky, .abi = .gnueabi },
+    .{ .name = "csky-linux-gnueabihf", .arch = .csky, .abi = .gnueabihf },
+    .{ .name = "loongarch64-linux-gnu", .arch = .loongarch64, .abi = .gnu },
+    .{ .name = "loongarch64-linux-gnusf", .arch = .loongarch64, .abi = .gnusf },
     .{ .name = "m68k-linux-gnu", .arch = .m68k, .abi = .gnu },
-    .{ .name = "mips-linux-gnu", .arch = .mips, .abi = .gnueabihf },
-    .{ .name = "mips-linux-gnu-soft", .arch = .mips, .abi = .gnueabi },
-    .{ .name = "mipsel-linux-gnu-soft", .arch = .mipsel, .abi = .gnueabi },
-    .{ .name = "mipsel-linux-gnu", .arch = .mipsel, .abi = .gnueabihf },
-    .{ .name = "mips64-linux-gnu-n64", .arch = .mips64, .abi = .gnuabi64 },
-    .{ .name = "mips64-linux-gnu-n32", .arch = .mips64, .abi = .gnuabin32 },
-    .{ .name = "mips64el-linux-gnu-n64", .arch = .mips64el, .abi = .gnuabi64 },
-    .{ .name = "mips64el-linux-gnu-n32", .arch = .mips64el, .abi = .gnuabin32 },
-    .{ .name = "powerpc-linux-gnu-soft", .arch = .powerpc, .abi = .gnueabi },
-    .{ .name = "powerpc-linux-gnu", .arch = .powerpc, .abi = .gnueabihf },
+    .{ .name = "mips-linux-gnueabi", .arch = .mips, .abi = .gnueabihf },
+    .{ .name = "mips-linux-gnueabihf", .arch = .mips, .abi = .gnueabi },
```

Comment:
I think they are swapped?

Reply:
Yep, good catch. And it also made me realize I can simplify this tool further.